airflow/providers/tableau/hooks/tableau.py

```diff
@@ -40,12 +46,12 @@ class TableauJobFinishCode(Enum):
 class TableauHook(BaseHook):
     """
     Connects to the Tableau Server Instance and allows to communicate with it.
-
```

Comment:
Please don’t remove this blank line, it’s crucial for formatting. (Same for other similar changes below.)